routers/repo/issue.go

```diff
@@ -615,6 +672,7 @@ func getBranchData(ctx *context.Context, issue *models.Issue) {
 		ctx.Data["BaseBranch"] = pull.BaseBranch
 		ctx.Data["HeadBranch"] = pull.HeadBranch
 		ctx.Data["HeadUserName"] = pull.MustHeadUserName()
+		ctx.Data["RequireSimpleMDE"] = true
```

Comment:
I think ths was for https://github.com/go-gitea/gitea/pull/8346#discussion_r365524323 but this is the wrong place :(